sdk/python/responsible-ai/responsibleaidashboard-diabetes-decision-making/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -1221,9 +1253,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
+   "display_name": "aml_rai_env",
```

Comment:
Could you please run python readme.py file? The notebook kernal should not be any random values, otherwise the notebooks cannot run on CI.
FYI: @balapv

Reply:
Done. Thanks for pointing this out. Is it possible to configure some linter to catch this?